server/methods/setUserActiveStatus.js

```diff
@@ -21,8 +21,35 @@ Meteor.methods({
 			});
 		}
 
-		setUserActiveStatus(userId, active, confirmRelenquish);
+		const user = Meteor.users.findOne({
+			_id: userId,
+		}, {
+			fields: {
+				_id: 1,
+				roles: 1,
+				active: 1,
+			},
+		});
 
+		// Check if user is the last ACTIVE admin and Prevent removing last user from admin role
+		const userIsAdmin = user.roles.indexOf('admin') > -1;
+		if (userIsAdmin) {
+			const adminCount = Meteor.users.find({
+				roles: {
+					$in: ['admin'],
+				},
+				active: true,
+			}).count();
+			if (adminCount === 1 && userIsAdmin && user.active) {
```

Comment:
Here, you just need the `adminCount === 1` condition. The `userIsAdmin` is assumed from the previous if, and the `user.active` shouldn't matter (since if the user is already disabled, it won't be an admin anyways).

Reply:
Sure i'll make the changes you suggested :)